index.js

```diff
@@ -1,3 +1,216 @@
+
+  const pBtn = document.createElement('button');
+  pBtn.setAttribute('type', 'button');
+  pBtn.setAttribute('id', project.id);
+  pBtn.classList.add('work-card-button');
+  pBtn.textContent = 'See Project';
+
+  pContent.appendChild(pBtn);
+
+  projectSection.appendChild(pCard);
+}
+
+projectCards.forEach(createCard);
+
+const modalContainer = document.getElementById('modal-container');
+
+function closeModal() {
+  modalContainer.classList.remove('show');
+}
+
+function createModal(id) {
+  // Search the project with the same id in the projects array
+  const project = projectCards.find((project) => Number(project.id) === Number(id));
+  const modal = document.createElement('div');
+  modal.setAttribute('id', 'modal');
+  modalContainer.appendChild(modal);
+
+  const modalHeaderContainer = document.createElement('div');
+  modalHeaderContainer.classList.add('modal-header-container');
+  modal.appendChild(modalHeaderContainer);
+
+  const modalHeader = document.createElement('h3');
+  modalHeader.classList.add('modal-header');
+  modalHeader.textContent = project.title;
+  modalHeaderContainer.appendChild(modalHeader);
+
+  const modalClose = document.createElement('div');
+  modalClose.classList.add('modal-close');
+  modalHeaderContainer.appendChild(modalClose);
+  modalClose.addEventListener('click', closeModal);
+
+  for (let i = 0; i < 3; i += 1) {
+    const closeSpan = document.createElement('span');
+    closeSpan.classList.add('line');
+    modalClose.appendChild(closeSpan);
+  }
+
+  const modalImg = document.createElement('img');
+  modalImg.classList.add('modal-image');
+  modalImg.setAttribute('src', project.img);
+  modalImg.setAttribute('alt', 'Project thumbnail');
+  modal.appendChild(modalImg);
+
+  const modalContent = document.createElement('div');
+  modalContent.classList.add('modal-content');
+  modal.appendChild(modalContent);
+
+  const modalP = document.createElement('p');
+  modalP.classList.add('modal-p');
+  modalP.textContent = project.description;
+  modalContent.appendChild(modalP);
+
+  const modalTags = document.createElement('ul');
+  modalTags.classList.add('tags-section');
+  modalContent.appendChild(modalTags);
+
+  project.tags.forEach((_element, index) => {
+    const tag = document.createElement('li');
+    tag.classList.add('tag');
+    modalTags.appendChild(tag);
+
+    const anchorTag = document.createElement('a');
+    anchorTag.setAttribute('href', '#');
+
+    anchorTag.textContent = project.tags[index];
+    tag.appendChild(anchorTag);
+  });
+
+  const modalBtnContainer = document.createElement('div');
+  modalBtnContainer.setAttribute('id', 'modal-btn-container');
+  modalContent.appendChild(modalBtnContainer);
+
+  const liveBtn = document.createElement('a');
+  liveBtn.setAttribute('id', 'live');
+  liveBtn.setAttribute('href', project.link);
+  liveBtn.textContent = 'See Live';
+  const liveIcon = document.createElement('img');
+  liveIcon.setAttribute('src', '/multimedia/live-icon.svg');
```

Comment:
- [ ] [OPTIONAL] Kindly make sure to change the src for image by removing the `/` from beginning of the path because if you upload your site the image will not appear. Also change it from all places in your portfolio.

Reply:
@omarsalem7 Done :heavy_check_mark: